solt/core/_data.py

```diff
@@ -362,21 +362,15 @@ class Keypoints(object):
     Parameters
     ----------
     pts : numpy.ndarray
-        Key points as an numpy.ndarray in (x, y) format.
-    frame : (n, ) list-like
+        Array of keypoints. If 2D, has to be in (x, y) format.
+    frame : (n, ) list-like of ints
```

Comment:
SOLT uses tuples by default for everything

Reply:
Tuple is list-like. This is solely for user convenience, so that the user does not need to worry whether to pass list or tuple. Internally, the type of `frame` is consistent.